Polyhedron/demo/Polyhedron/Plugins/Surface_mesh_deformation/Scene_edit_polyhedron_item.cpp

```diff
@@ -244,7 +244,8 @@ void Scene_edit_polyhedron_item_priv::initializeBuffers(CGAL::Three::Viewer_inte
         item->buffers[Facet_normals].release();
         item->vaos[Facets]->release();
         program->release();
-        delete vertices;
+        if(offset.norm() !=0)
+          delete vertices;
```

Comment:
Why was `vertices` created on the heap anyway? If that `std::vector` is meant to be temporary, then just create it on the stack!
```C++
std::vector<GLdouble> vertices;
if(offset.norm() !=0) {
  vertice.resize(positions.size()));
  // ...
}